### DISCUSSION TITLE
Run on a specified port instead of just 3000

### DISCUSSION THREAD

**Discussion by ashokrags** [gitpod-io/openvscode-server#248] [General] [answered]
hi,
  This is great and i would like to be able to run either the linux executable or the docker container on a random port instead of port 3000. I do realize that docker could be used for a random port, but I am planning to run this with singularity on a HPC cluster which does not allow for port mapping.
   Any thoughts/suggestions will be much appreciated.
     Cheers
   Ashok

**Comment by jeanp413** [comment, marked as answer]
@ashokrags to run the server on a specific port just pass the `--port 1234` (`0` for a random port) argument to the `server.sh` script
